_posts/0000-01-02-joeyglasgow.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Is this where I am supposed to write
Use the left arrow to go back!

Reply:
I think so